client/src/views/PeerReview/PeerReview.js

```diff
@@ -129,8 +129,8 @@ const PeerReview = props => {
             review.employee_under_review.lastname}
         </span>,
         review.employee_reviewing.firstname +
-          ' ' +
-          review.employee_reviewing.lastname,
+        ' ' +
+        review.employee_reviewing.lastname,
```

Comment:
I think we need to use template literal here. Is it possible to use template literal?

Reply:
did in PR #31